posthog/templates/redis/values.html

```diff
@@ -0,0 +1,67 @@
+{% extends "admin/base_site.html" %}
```

Comment:
This is mostly copied from the django changelist template, modified to fit our view.